### DISCUSSION TITLE
i can`t install osu!lazer. Authorization Required

### DISCUSSION THREAD

**Discussion by Channel9832** [ppy/osu#18633] [Q&A]
Today I tried to download osu!Lazer, but at one point it asks me for a username and password to access, at first I thought it asked me for the name and password of my PC, but it told me that it is incorrect, I tried the one from github and also It was wrong, does anyone know what password it is?

error code: Authorization Required

**Comment by peppy** [comment]
It sounds like you are downloading it from the wrong place.

Please use the links at https://github.com/ppy/osu#running-osu 

Also *please* don't type your PC login into random dialogs, it's very dangerous. Only put your PC login into something which you know is the operating system itself.
